cairo/ethereum/cancun/state.cairo

```diff
@@ -510,18 +510,38 @@ func account_has_code_or_nonce{poseidon_ptr: PoseidonBuiltin*, state: State}(
     return res;
 }
 
+// @notice Returns all (address, key) pairs for a given address.
+// This is a temporary fix specifically made for the `account_has_storage` function.
+// We need to make a deeper refactor once we have a sound way to handle storage keys.
+func get_storage_keys_for_address{dict_ptr: DictAccess*}(
```

Comment:
nit: put both this one and get_keys_for_address_prefix next to each other in this file or in legacy.utils.dict.